source/components/ansible-setup.rst

```diff
@@ -59,6 +59,19 @@ Allan have access to it. Stash the password in file named
 You will also need sudo access on the hosts you want to change if that change
 requires privilege escalation.
 
+.. note::
+    The ``ansible-conda`` submodule should be copied locally using the
+    following command:
+
+    .. code-block:: bash
```

Comment:
This needs a blank line. The `git submodule ...` line is being interpreted as a setting on `code-block` rather than content of the block.

From the Travis build:

```
/home/travis/build/NSLS-II/docs/source/components/ansible-setup.rst:66:Error in "code-block" directive:
maximum 1 argument(s) allowed, 6 supplied.
```

Reply:
done